src/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

```diff
@@ -28,48 +28,86 @@
 namespace Microsoft.Identity.Client.AppConfig
 {
     /// <summary>
-    ///     Options object with string values loadable from JSON configuration (as in an asp.net configuration scenario)
+    /// Options object with string values loadable from a configuration file 
```

Comment:
Will this really load from a config file, since this is an abstract class.

Reply:
thanks @bgavrilMS. Addressed